mozlog/mozlog.go

```diff
@@ -1,3 +1,4 @@
+// Package mozlog is an implementation of the mozlog logging format.
```

Comment:
We keep this code in its own package because it overrides the logging library and we auto-load it in the `main.go` file (via `init()` defined here and the `_` import thingy)